### DISCUSSION TITLE
1.3.0b2 -> potential 1.3.0 release

### DISCUSSION THREAD

**Discussion by snmishra** [snmishra/ftd2xx#49] [Announcements]
- Fixed several bugs with ctypes (thanks @krister-ts)
- Remove aio import in sync version

<hr /><em>This discussion was created from the release <a href='https://github.com/snmishra/ftd2xx/releases/tag/v1.3.0-beta.2'>ctypes fixes</a>.</em>

**Comment by gek47** [comment]
On linux I have tested the following method no problems, no testing on windows 
Thanks for all your hard work

setVIDPID
openEx  by Serial number 
createDeviceInfoList
getDeviceInfoDetail

**Reply by snmishra** [reply]
@gek47 Appreciate the testing. I'm going to release as 1.3.0 🤞

**Comment by gek47** [comment]
sorry I should have included
setBaudRate
setTimeouts
purge
getStatus
close
read
write
